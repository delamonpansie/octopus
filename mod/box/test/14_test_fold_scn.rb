#!/usr/bin/ruby

$: << File.dirname($0) + '/lib'
require 'run_env'

RunEnv.connect_eval do
  12.times {|i| insert [i,i]}

  puts `./octopus --fold 2 -vALL=-2 2>/dev/null`
end
