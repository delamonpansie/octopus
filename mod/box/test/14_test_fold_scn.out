# box.insert([0, 0])
1

# box.insert([1, 1])
1

# box.insert([2, 2])
1

# box.insert([3, 3])
1

# box.insert([4, 4])
1

# box.insert([5, 5])
1

# box.insert([6, 6])
1

# box.insert([7, 7])
1

# box.insert([8, 8])
1

# box.insert([9, 9])
1

# box.insert([10, 10])
1

# box.insert([11, 11])
1

CRC: 0x0ebc74b5
